src/commands/monitors/run.rs

```diff
@@ -25,13 +28,6 @@ pub fn make_command(command: Command) -> Command {
                 .default_value("production")
                 .help("Specify the environment of the monitor."),
         )
-        .arg(
```

Comment:
We cannot remove flags in minor releases, as it will break users using them. We need to hide them instead.